oml/imce.jpl.nasa.gov/discipline/power/pel.oml

```diff
@@ -0,0 +1,133 @@
+@dc:rights="""Copyright 2010-2019, by the California Institute of Technology. ALL RIGHTS RESERVED. United States Government Sponsorship acknowledged. Any commercial use must be negotiated with the Office of Technology Transfer at the California Institute of Technology.
+
+This software may be subject to U.S. export control laws. By accepting this software, the user agrees to comply with all applicable U.S. export laws and regulations. User has the responsibility to obtain export licenses,a or other export authority as may be required before exporting such information to foreign countries or providing access to foreign persons."""
+@owl:versionInfo="""$Revision$"""
+@dc:creator="""Flight Systems Engineering, Integration, & Test Section"""
+@dc:date="""$Date$"""
+@dc:type="""Discipline"""
+@dc:title="""PEL Ontology"""
+@dc:description="""The PEL Ontology defines concepts and properties for describing power equipment lists."""
+open terminology <http://imce.jpl.nasa.gov/discipline/power/pel> {
+	
+	extends <http://imce.jpl.nasa.gov/foundation/behavior/behavior>
+	
+	extends <http://iso.org/iso-80000-2>
+	
+	extends <http://iso.org/iso-80000-4>
+	
+	@dc:description="""TBD"""
+	@rdfs:label="""Volt Parameter"""
+	concept VoltParameter
+	
+	@dc:description="""TBD"""
+	@rdfs:label="""Watt Parameter"""
+	concept WattParameter
+	
+	@dc:description="""TBD"""
+	@rdfs:label="""Percent Parameter"""
+	concept ContingencyParameter
+	
+	// TODO Move to a power ontology?
+	@dc:description="""TBD"""
+	@rdfs:label="""Watt Codomain"""
+	concept VoltCodomain
+	
+	@dc:description="""TBD"""
+	@rdfs:label="""Watt Codomain"""
+	concept WattCodomain
+	
+	@dc:description="""TBD"""
+	@rdfs:label="""Percent Codomain"""
+	concept PowerContingencyCodomain
+	
+	@dc:description="""TBD"""
+	@rdfs:label="""Constant Value Constraint"""
+	concept ConstantValueConstraint
+	
+	@dc:description="""TBD"""
+	@rdfs:label="""MEV Constraint"""
+	concept MEVConstraint
+	
+	VoltParameter extendsConcept behavior:Parameter
+	
+	WattParameter extendsConcept behavior:Parameter
+	
+	ContingencyParameter extendsConcept behavior:Parameter
+	
+	VoltCodomain extendsConcept behavior:ScalarCodomain
+	
+	WattCodomain extendsConcept behavior:ScalarCodomain
+	
+	PowerContingencyCodomain extendsConcept behavior:ScalarCodomain
+	
+	ConstantValueConstraint extendsConcept behavior:Constraint
+	
+	MEVConstraint extendsConcept behavior:Constraint
+	
+	@dc:description="""TBD"""
+	@oml:hasPropertyLabel="""constrainsCBEValue"""
+	@oml:hasReificationLabel="""ConstrainsCBEValue"""
+	reifiedRelationship ConstrainsCBEValue {
+		functional
+		essential
+		unreified = constrainsCBEValue
+		source = MEVConstraint
```

Comment:
Do you mean CBEConstraint here?

Reply:
No, MEVConstraint - the idea here is to have an "MEVConstraint" that constrains CBE, MEV and contingency value.

Reply:
I don't understand that. I guess we'll discuss it in review.

Reply:
MEVConstraint is supposed to represent the constraint that relates CBE, MEV and contingency values. I.e.: `MEV = CBE * (1 + contingency)`. My understanding from the conversations that I've had with you and Nic in the past is that this is how constraints should / could be captured in OML.

Reply:
Pushed some changes after our discussion today. Please re-review, and resolve if you are satisfied with the changes